### PR TITLE
more explanation for lr_schedular

### PR DESCRIPTION
**Pull request by wenyushi**
I plan to push more explanation for lr policies.
The affected file is dlpy/lr_scheduler.py.

### CODE REVIEW

File: dlpy/lr_scheduler.py [hunk 1]
```diff
@@ -181,6 +185,7 @@ def __init__(self, learning_rate, power):
 class ReduceLROnPlateau(FCMPLR):
     """
     Reduce learning rate on plateau learning rate scheduler
+    Reduce learning rate when loss has stopped improving for a patience number of epochs.
```
Comment (ethem-kinginthenorth) [inline dlpy/lr_scheduler.py]: for certain number of epochs (I am assuming this is set by a parameter or you decide right)?

Comment (wenyushi) [reply dlpy/lr_scheduler.py]: yes, correct

Comment (ethem-kinginthenorth) [reply dlpy/lr_scheduler.py]: would you please update the comment. thanks.

**Comment by ethem-kinginthenorth** [comment]
thanks

**Pull request merged**